docs/installation-with-docker.md

```diff
@@ -28,13 +28,12 @@ $ cd scalar-samples
 ## Docker login
 
 `docker login` is required to start the Scalar DL Docker image. Because the
-[`scalar-server`](https://hub.docker.com/r/scalarlabs/scalar-server/) repository
-on Docker Hub is currently private, your Docker account needs to be listed as a
-collaborator of the repository. Ask a person in charge to get your account
-ready.
+[`scalar-server`](https://github.com/orgs/scalar-labs/packages/) repository
+on GitHub Container Registry is currently private, your GitHub account needs to be set with permissions to access the container images.
+Ask a person in charge to get your account ready.
 
 ```
-$ docker login
+$ docker login ghcr.io
 ```
 
 ## Start up the server
```

Comment:
```suggestion
## Start up Scalar DL Ledger
```

Reply:
Can you also update the sentence below to this?

```The following command starts up Scalar DL Ledger along with the back-end
Cassandra server in the Docker containers. At the first time you run the
command, the required Docker images will be downloaded from Docker Hub.```

Reply:
Fixed in 5df8523.

Reply:
https://github.com/scalar-labs/scalardl/pull/9/files#diff-2e9d792c66c60f65e3e5367a87f6145398069017f0aa900529fc19ffaacfd5dcL42-R42

"Docker Hub" should be changed, and the first line should not be removed.

Reply:
Thanks! Fixed in 05ebbda